src/Raven.Server/Documents/Handlers/AttachmentHandler.cs

```diff
@@ -102,6 +102,7 @@ public Task GetPost()
                         HttpContext.Response.Headers.Remove("Content-Type");
                 }
                 HttpContext.Response.Headers["Content-Hash"] = attachment.Base64Hash.ToString();
+                HttpContext.Response.Headers["Content-Size"] = attachment.Stream.Length.ToString();
```

Comment:
Maybe change both? Content-Hash -> Attachment-Hash and Content-Size to Attachment-Size?